SU-SOE/stanford_soe_helper
# SOE-2270 7.x 2.x tweaks-03 11-21 for magazine articles

# READY FOR REVIEW

# Summary
- This implements CSS fixes for magazine articles.

# Needed By (Date)
- When possible

# Criticality
- Fixed on prod, need to put into code.

# Steps to Test
- Switch to this branch
- Navigate to: /magazine/article/new-textile-can-keep-people-comfortable-whatever-weather
- Use https://docs.google.com/spreadsheets/u/1/d/1O7VSJD25zNH722Fbg4XcHl3M6KkAZAGKtzRCkU9M45c/edit#gid=0 to verify

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2270

## Related PRs

## More Information

## Folks to notify
Kerri


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
